docs/source/1_getting_started/hello_world.rst

```diff
@@ -100,11 +95,11 @@ Now we must include our node set to the information model of the application in
     <NodeSetFile>@CONF_DIR@/HelloWorldNodeSet.xml</NodeSetFile>
   </InformationModel>
 
-This might seem quite complicated, but actually you won't need to make your XML node sets manually. You can use our `OPC UA Designer`_ to make this process easier. 
-However we do everything ourselves, so that we can learn some basic OPC UA conceptions. If you are already familiar to the protocol, just skip the rest of the section. 
+This might seem quite complicated, but actually you won't need to make your XML node sets manually. You can use our `OPC UA Designer`_ to make this process easier.
+However we do everything ourselves, so that we can learn some basic OPC UA conceptions. If you are already familiar to the protocol, just skip the rest of the section.
 
-*OPC UA Information* model is split into *namespaces*. Each of them must have its *namespace index*. Index 0 is reserved for the OPC UA standard namespace, where all standard types are described. 
-In our application it is stored in **Opc.Ua.NodeSet.xml**. The application can't work without it, so we need to include this file in our configuration. 
+*OPC UA Information* model is split into *namespaces*. Each of them must have its *namespace index*. Index 0 is reserved for the OPC UA standard namespace, where all standard types are described.
```

Comment:
OPC UA defines two namespaces. The namespace with the namspace index 0 is the standard namespace. The namespace with the namespace index 1 is the namespace of the opc ua application server. These two namespaces must not be used by the application.

The OPC UA server currently does not offer the namespace 1 yet. This is a bug.

Reply:
I see. When we fix the bug, I'll update the document.